config/scripts/postinstall.js

```diff
@@ -13,18 +13,19 @@ const packageJson = JSON.parse(
 )
 
 if (!packageJson.msw || !packageJson.msw.workerDirectory) {
-  return
+  process.exit()
 }
 
 // 2. Check if the worker directory is an existing path.
 const { workerDirectory } = packageJson.msw
 const absoluteWorkerDirectory = path.resolve(parentPackageCwd, workerDirectory)
 
 if (!fs.existsSync(absoluteWorkerDirectory)) {
-  return console.error(
+  console.error(
     `[MSW] Failed to automatically update the worker script at "%s": given path does not exist.`,
     workerDirectory,
   )
+  process.exit()
```

Comment:
I think this is a bad idea. These cases are meant to halt early but don't exit the process (`npm install`).